src/main/java/edu/fiuba/algo3/Weather.java

```diff
@@ -0,0 +1,7 @@
+package edu.fiuba.algo3;
+
+public abstract class Weather extends Special {
+    public Weather(String name, String description) {
+        super(name, description);
+    }
```

Comment:
If you don't have extra logic(you only call super), you don't need to define it, the class will use the constructor of Mother class.